settings.gradle

```diff
@@ -43,7 +43,8 @@ include ':jgiven-core',
         ':jgiven-java8-tests',
         ':jgiven-junit5',
         ':jgiven-spring-junit5',
-        ':jgiven-gradle-plugin'
+        ':jgiven-gradle-plugin',
+        ':jgiven-timing'
```

Comment:
I'm not sure if we should include the project for releases... Maybe you should make this a conditional include

Reply:
if you have  https://github.com/TNG/JGiven/pull/691/files#r669678733
Then you could do
```
 if (timingEnabled){
  include ':jgiven-timing'
}
```